package.json

```diff
@@ -5,61 +5,61 @@
 	"private": false,
 	"description": "Fast 3kb React-compatible Virtual DOM library.",
 	"main": "dist/preact.js",
-	"module": "dist/preact.module.js",
+	"module": "dist/preact.module.mjs",
```

Comment:
This isn't safe to change outside of a semver major. A lot of tools/versions of tools can consume `"module"` but don't all support `.[cm]js` extensions.

A few months ago [`immer` stumbled into an issue making this change](https://github.com/immerjs/immer/issues/937), discovering that Metro didn't support `.mjs`. Only as of last month [has that been rectified](https://github.com/facebook/metro/issues/535#issuecomment-1198071838).

While I doubt Metro would be an issue for us, that just goes to show that support is not as widespread as we may hope. Node isn't going to consume `"module"` so it shouldn't need to adhere to Node's file extension semantics.